iib/web/api_v1.py

```diff
@@ -705,3 +707,44 @@ def merge_index_image():
 
     flask.current_app.logger.debug('Successfully scheduled request %d', request.id)
     return flask.jsonify(request.to_json()), 201
+
+
+@api_v1.route('/builds/create-empty-index', methods=['POST'])
+@login_required
+def create_empty_index():
+    """
+    Submit a request to create an index image without bundles.
+
+    Note: Any duplicate bundle will be removed from payload.
+
+    :rtype: flask.Response
+    :raise ValidationError: if required parameters are not supplied
+    """
+    payload = flask.request.get_json()
+    if not isinstance(payload, dict):
+        raise ValidationError('The input data must be a JSON object')
+
+    request = RequestCreateEmptyIndex.from_json(payload)
+    db.session.add(request)
+    db.session.commit()
+    messaging.send_message_for_state_change(request, new_batch_msg=True)
+
+    args = [
+        payload['from_index'],
+        request.id,
+        payload.get('binary_image', None),
+        payload.get('labels', None),
```

Comment:
You don't have to specify `None` as the default value. It defaults to none if nothing is specified.